frontend/src/pages/Dashboard.js

```diff
@@ -36,6 +36,16 @@ const Dashboard = () => {
         }
     )
 
+    const [allUsers, updateAllUsers] = useReducer(
```

Comment:
useReducer is a bit more complicated (for multiple pieces of coupled state) and unnecessary here. can just useState